src/Our.Umbraco.Ditto/Extensions/PublishedContentExtensions.cs

```diff
@@ -1,4 +1,9 @@
-﻿namespace Our.Umbraco.Ditto
+﻿using System.Collections;
+using Newtonsoft.Json;
+using Our.Umbraco.Ditto.Models.Archetype;
+using Our.Umbraco.Ditto.Models.Archetype.Abstract;
```

Comment:
I got stung by this on my first commit. We're keeping a flat namespace structure. `Our.Umbraco.Ditto`